driver.go

```diff
@@ -39,22 +39,22 @@ func initDriver() error {
 	ret = api.SQLSetEnvUIntPtrAttr(drv.h, api.SQL_ATTR_ODBC_VERSION, api.SQL_OV_ODBC3, 0)
 	if IsError(ret) {
 		defer releaseHandle(drv.h)
-		return NewError("SQLSetEnvUIntPtrAttr", drv.h)
+		return NewError("SQLSetEnvUIntPtrAttr(SQL_ATTR_ODBC_VERSION, SQL_OV_ODBC3)", drv.h)
```

Comment:
These errors are going to be arisen on ODBC Driver Manager issues.
This is not supposed to happen for Microsoft SQL Server, of course, so many users will not see anything of that.

However, several commercial ODBC Drivers embed a driver manager themselves, that during the installation might replace unixODBC on Linux, without full compatibility with unixODBC. 
In this case, this kind of errors might be arisen, providing an helpful hint about what's is wrong in the ODBC driver integration/configuration.

That was very helpful to me for understanding what was wrong the integration of Teradata Vantage.
It might be helpful to any other users that would try to use a new ODBC driver.

Having said that, you are right, I implemented a simple workaround to distinguish the kind of error at init execution. This workaround breaks the semantic of Error.APIName, and, as you correctly observed, maybe it is not worth the trouble extending Error type to support the arguments, too.

Let try to be effective and simple. 
What do you think to refactor Error.APIName to Error.APIDesc or Error.APIInfo (or any other appropriate field name)?
In this way, everyone can be happy.
I can do that.

Reply:
I am still not convinced that it is helpful. But, since you insist, lets add private Error.apiParam  string. And it will only be displayed, if it is not empty.

I don't want anyone use new filed directly, so we can change the code if we want to.